### DISCUSSION TITLE
Brainsight MNI export

### DISCUSSION THREAD

**Discussion by Evirtal** [simnibs/simnibs#470] [9. Results and analysis] [answered]
Hi, 
I am using brainsight but have a whole set of subjects that used a T1 different from the one that comes from simnibs. I was wondering if it would be possible to import the TMS list with mni coordinate and then use the mni2subject function afterward. Also is it possible to have the import TMS list function of brainsight in matlab?
Kind regards

**Comment by TorgeW** [comment, marked as answer]
Yes you can try this, however please not that the transformations from subject space to mni space in brainsight and simnibs rely on different methods so that we don't know about the accuracy of this approach.

**Reply by Evirtal** [reply]
Ok thank you, is there a matlab function to import TMS list to simnibs? I do not think I have found any
Regarding the accuracy, I'll add a false target onto a gyri to check if it stays in place

**Reply by bjornsig** [reply]
no, simnibs does not have a function to import the tms list written out by brainsight. however, you can try the python function which are included in simnibs.
